nin/dasBoot/BEATBEAN.js

```diff
@@ -5,8 +5,9 @@
     BEAN = 0;
     BEAT = false;
 
-    var subdivision = 6;
-    var beatsPerMinute = 110 * subdivision;
+    var musicInfo = PROJECT.get('music', {});
+    var subdivision = musicInfo.subdivision || 6;
+    var beatsPerMinute = (musicInfo.bpm || 110) * subdivision;
```

Comment:
Slightly inaccurate variable name. BPM (Beats Per Minute) is already the number that is set to 110 in this example. The variable holding the number 110 * 6 (in this example) should be called something different.
How about `subBeatsPerMinute` instead?